src/SFA.DAS.Commitments.Api/Controllers/ProviderController.cs

```diff
@@ -97,5 +99,15 @@ public ProviderController(ProviderOrchestrator providerOrchestrator)
 
             return StatusCode(HttpStatusCode.NoContent);
         }
+
+        [Route("{providerId}/commitments/{commitmentId}/apprenticeships/bulk")]
+        [Authorize(Roles = "Role1")]
+        public async Task<IHttpActionResult> PostBulkUpload(long providerId, long commitmentId, IList<Apprenticeship> apprenticeships)
+        {
+            // TODO: What should we return to the caller? list of urls?
+            await _providerOrchestrator.CreateApprenticeships(providerId, commitmentId, apprenticeships);
```

Comment:
perhaps same approach as in ApprenticeshipEventsController.cs where we return the location of the "parent collection"